src/pages/public_registration/action_accordions/SupportAccordion.tsx

```diff
@@ -40,15 +40,13 @@ export const SupportAccordion = ({ registration, supportTicket, addMessage, refe
   });
 
   const createSupportTicket = async (message: string) => {
-    const createTicketResponse = await createTicket(registration.id, 'GeneralSupportCase', true);
+    const createTicketResponse = await createTicket(registration.id, 'GeneralSupportCase', false, message);
```

Comment:
Ser param er endret fra `true` til `false` her, mulig den der var en workaround for å få det til å virke og at den nå kan fjernes?
En annen notis er at message kanskje er mer vanlig enn det bool-flagget, sånn at `message` burde komme før som parameter også